modules/metering-viewing-report-results.adoc

```diff
@@ -16,38 +16,55 @@ Reports can be retrieved as `JSON`, `CSV`, or `Tabular` formats.
 
 . Change to the `openshift-metering` project:
 +
+[source,terminal]
 ----
 $ oc project openshift-metering
 ----
 
 . Query the reporting API for results:
 
-.. Get the route to the `reporting-api`:
+.. Create a variable for the metering `reporting-api` route then get the route:
 +
+[source,terminal]
 ----
 $ meteringRoute="$(oc get routes metering -o jsonpath='{.spec.host}')"
+----
++
+[source,terminal]
+----
 $ echo "$meteringRoute"
 ----
 
 .. Get the token of your current user to be used in the request:
 +
+[source,terminal]
 ----
 $ token="$(oc whoami -t)"
 ----
 
+.. Set `reportName` to the name of the `Report` you created:
```

Comment:
Does `report` here need markup? Seems like most other instances of _Report_ does not use markup.